test/quantization/test_numeric_suite.py

```diff
@@ -22,3 +26,49 @@ def test_compare_weights(self):
         self.assertEqual(len(weight_dict), 1)
         for k, v in weight_dict.items():
             self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+        annotated_conv_bn_relu_model = AnnotatedConvBnReLUModel().eval()
+        annotated_conv_bn_relu_model.fuse_model()
+        quantized_annotated_conv_bn_relu_model = quantize(
+            annotated_conv_bn_relu_model, default_eval_fn, self.img_data
+        )
+        weight_dict = compare_weights(
+            annotated_conv_bn_relu_model.state_dict(),
+            quantized_annotated_conv_bn_relu_model.state_dict(),
+        )
+        self.assertEqual(len(weight_dict), 1)
+        for k, v in weight_dict.items():
+            self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+    def test_compare_model_stub(self):
+        r"""Compare the output of quantized conv layer and its float shadow module
+        """
+        # eager mode
+        annotated_conv_model = AnnotatedConvModel().eval()
+        quantized_annotated_conv_model = quantize(
+            annotated_conv_model, default_eval_fn, self.img_data
+        )
+        data = self.img_data[0][0]
+        module_swap_list = [torch.nn.Conv2d]
+        ob_dict = compare_model_stub(
+            annotated_conv_model, quantized_annotated_conv_model, module_swap_list, data
+        )
+        self.assertEqual(len(ob_dict), 1)
+        for k, v in ob_dict.items():
+            self.assertTrue(v["float"].shape == v["quantized"].shape)
+
+        annotated_conv_bn_relu_model = AnnotatedConvBnReLUModel().eval()
+        annotated_conv_bn_relu_model.fuse_model()
+        quantized_annotated_conv_bn_relu_model = quantize(
+            annotated_conv_bn_relu_model, default_eval_fn, self.img_data
+        )
+        module_swap_list = [nn.intrinsic.modules.fused.ConvReLU2d]
+        ob_dict = compare_model_stub(
```

Comment:
Please add an additional test for swapping a user defined module here. For example
```python
class ModelWithSubModules(torch.nn.Module):
    def __init__(self):
        super(ModelWithSubModules, self).__init__()
        self.qconfig = default_qconfig
        self.mod1 = subModule()
        self.conv = torch.nn.Conv2d(...)
        self.quant = QuantStub()
        self.dequant = DeQuantStub()

def forward(self, x):
     x = self.quant(x)
     x = self.mod1(x)
     x = self.conv(x)
     x= self.dequant(x)
    return x
```
 The test should add a stub only to mod1